index.html

```diff
@@ -0,0 +1,26 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <title>Calculator</title>
+    <link rel="stylesheet" href="./assets/css/style.css">
+</head>
+<body>
+    <div class="wrapper">
```

Comment:
The end style of the page is lacking some styling from the provided design.

Reply:
Can you explain in more detail what you mean?

Reply:
- the font family and size are not like in the design
- the body bg color is not like in the design
- the add button color is not the same as the one in the design
- add/even bg color is not like in the design
- the wrapper shadow is not like in the design

You can https://www.figma.com/file/GPLBZNBQqu4WlRvoB9kbZt9c/Untitled?node-id=1%3A3 open the figma design and select the most right tab "Code" in the right panel. when selecting elements from the desing there you can see suggested css for each.

Reply:
Thank you! I think that I correct it. :)